.azure/azure-pipelines.ci.yml

```diff
@@ -150,6 +150,13 @@ stages:
       platform: windows
       tls: schannel
       logProfile: Full.Light
+  - template: ./templates/run-bvt.yml
+    parameters:
+      pool: MsQuic-Win-Latest
+      platform: windows
+      tls: schannel
+      logProfile: Full.Light
+      codeCoverage: true
```

Comment:
This is just a temporary attempt to get AZP to show something. As we discussed before, it's probably better not to need a separate run for this.

Reply:
we're going to keep them separate for now since it appears that the code coverage tools eat crashes.